src/graph.jl

```diff
@@ -96,3 +96,14 @@ function get_attributes(raw_graph::Dict{String, Any},
     end
     return attributes
 end
+
+function get_population_of_subgraph(graph::BaseGraph, nodes::Set{Int})::Int
+    """ Returns the population of the subgraph induced by `nodes`.
+        Potential TODO: `nodes` is a set, and iterating over a set is slower than an array.
+    """
+    total_pop = 0
```

Comment:
Could this be replaced with a one-liner, e.g. `sum(graph.populations[node] for node in nodes)`?

Reply:
lol yes i will do that